File: src/main/java/net/bingecraft/bloodmoon/BloodmoonClock.java
package net.bingecraft.bloodmoon;

import org.bukkit.Difficulty;

public final class BloodmoonClock {
    public Difficulty getDifficulty(long tick) {
        return (tick + 71000L) % 72000L < 59000L
                ? Difficulty.PEACEFUL
                : Difficulty.HARD;
    }
}

File: src/main/java/net/bingecraft/bloodmoon/BloodmoonClock.kt
package net.bingecraft.bloodmoon;

import org.bukkit.Difficulty

class BloodmoonClock {
  fun getDifficulty(tick: Long): Difficulty {
    return if ((tick + 71000L) % 72000L < 59000L) Difficulty.PEACEFUL
    else Difficulty.HARD
  }
}
